anaInfo.py

```diff
@@ -17,6 +19,9 @@
         "scurve":("SCurveData.root","scurveTree"),
         "scurveAna":("SCurveData/SCurveFitData.root","scurveFitTree"),
         "threshold":("ThresholdScanData.root","thrTree"),
```

Comment:
I think you forgot to remove line 21.

Reply:
Hmm, I may have left it to ensure that legacy behaviour remained, but can't remember; for sure with the check against valid options, it will never be used, so can probably be removed